src/main/java/org/spongepowered/api/text/Text.java

```diff
@@ -971,6 +975,162 @@ public static Text of(Object... objects) {
     }
 
     /**
+     * Returns the Text value for the specified key in the specified
+     * {@link Dictionary} for the specified {@link Locale}.
+     *
+     * @param dictionary Dictionary to search
+     * @param key Key to search for
+     * @param locale Locale target
+     * @return Localized text
+     */
+    public static Text get(Dictionary dictionary, String key, Locale locale, String def) {
```

Comment:
Hmm yeah, I see your point but I personally like this from a design standpoint and since these are static the "representation" of the Text object is the same anyways. But the location of these methods will definitely be worth discussing down the line a bit.